variables/_forms.pcss

```diff
@@ -6,7 +6,7 @@
 	--form-color-background: var(--color-background);
 	--form-color-border-default: var(--color-text-primary);
 	--form-color-border-active: var(--color-accent-secondary);
-	--form-color-border-secondary: var(--color-text-disabled);
+	--form-color-border-secondary: var(--color-border-tertiary);
```

Comment:
changing this will affect many styles that use this downstream. TEC uses it once, FB uses it once, but ET uses it many times. Add another variable here or just use `--color-border-tertiary` directly.

Reply:
I thought the (secondary) point was to bring them all into WCAG compliance?

Reply:
I just checked the ET uses and it _should_ be fine. I'd check just to be sure though.

Reply:
All of the uses of that var are for toggles/sliders - and the only _actual_ use I could find is for the `kitchen sink`.